src/ILCompiler.Compiler/src/Compiler/JitHelper.cs

```diff
@@ -168,6 +168,33 @@ public static void GetEntryPoint(TypeSystemContext context, ReadyToRunHelper id,
                     methodDesc = context.GetHelperEntryPoint("MathHelpers", "ULMulOvf");
                     break;
 
+#if ARM
```

Comment:
We don't do `#if` in the compiler codebase because it hinders cross-targeting. Could you make these a `if (context.Target.Architecture ==...)` runtime check?